tests/test_alertmanager.py

```diff
@@ -61,3 +62,29 @@ def test_alertmanager_rfc1918():
         )
         for value in item
     )
+
```

Comment:
All of this content in test_alertmanager.py can be removed since it is done in test_global_storageclass.py

Reply:
@danielhoherd  - test cases removed from alert manager.

Reply:
If it's all good we can merge this to master.